README.md

```diff
@@ -115,38 +120,39 @@ You can see collection of all configuration files at *$TIZENRT_BASEDIR/build/con
 To check all pre-defined configurations, type as follows:
 
 ```bash
-./configure.sh --help
+./tools/configure.sh --help
 ```
 
-##### 1.1 Additional Configuration (optional)
-
-After basic configuration by [1. Configuration](#1-configuration), you can additionally modify your configuration with *menuconfig*.
+Additionally, you can modify your configuration with *menuconfig*.
 
 ```bash
 ./dbuild.sh menuconfig
 ```
 
-**Note**: In Docker environment, ```make menuconfig``` command from other README files should be replaced with this command.  
-          ```make menuconfig``` applies only in [Manual Setup Build Environment](docs/HowToSetEnv.md).
+>**Note**
+>In Docker environment, ```make menuconfig``` command from other README files should be replaced with this command.
+>```make menuconfig``` applies only in [Manual Setup Build Environment](docs/HowToSetEnv.md).
 
 #### 2. Compilation
-
+There are two aspects to compilation, namely Build and Clean, which are described as follows:
+##### Build
+To compile, simply type the following:
 ```bash
 ./dbuild.sh
 ```
 
-Built binaries are located in *$TIZENRT_BASEDIR/build/output/bin*.
+After compilation, built binaries will be located in *$TIZENRT_BASEDIR/build/output/bin*.
 
-##### 2.1 Clean
+##### Clean
 
-There are two types of clean commands, clean and distclean.
+There are two types of clean commands, *clean* and *distclean*.
 
 ```bash
 ./dbuild.sh clean
 ```
```

Comment:
Could you check here for indentation?

Reply:
They are not numbered lists as such. However, if you prefer numbering them as 1. clean and 2. distclean, then the indentation will become necessary, in line with the general suggestion.